docs/project/README.md

```diff
@@ -1,45 +1,31 @@
 ---
-title: The IPFS project
-description: Learn about the history, roadmap, current status and more for IPFS, the InterPlanetary File System.
+title: The Filecoin Project
+description: Learn about the history, roadmap, current status and more for Filecoin
 ---
-# TODO: REPLACE CONTENT
+# The Filecoin Project
 
-# The IPFS project
+Curious about how it all got started, or where we're headed? Learn about the history, current state, and future trajectory of the Filecoin project here.
 
-Curious about how it all got started, or where we're headed? Learn about the history, current state, and future trajectory of the IPFS project here.
+## History of Filecoin
 
-## History of IPFS
-
-Want to know how it all began? Learn the [history of the IPFS project](/project/history/).
+Want to know how it all began? Learn the [history of the Filecoin project](/project/history/).
 
 ## Roadmap
 
-See the overall roadmap of [IPFS project requirements](https://github.com/ipfs/ipfs/blob/master/REQUIREMENTS.md) from current state to maturity.
-
-## Implementation status
+The latest timing information is available in the [Filecoin Mainnet Roadmap](https://app.instagantt.com/shared/s/1152992274307505/latest).
 
-Want to know the current state of your favorite IPFS feature? See the [current implementation status](https://github.com/ipfs/ipfs/blob/master/IMPLEMENTATION_STATUS.md) for go-ipfs and js-ipfs, listed by commands and endpoints.
+## Filecoin specification
 
-## IPFS specifications
-
-[Technical specifications](https://github.com/ipfs/specs) for the IPFS protocol and its associated subsystems.
+[Technical specification](https://github.com/filecoin-project/specs) for the Filecoin protocol and its associated subsystems.
 
 ## Research
 
-Learn more about the exploratory research work and prototyping being done for inclusion in IPFS by exploring our [research repo on GitHub](https://github.com/ipfs/research).
-
-## IPFS team org planning
-
-See how IPFS team planning, management and coordination happens, and learn how to take part in the weekly open call, by visiting the [IPFS team management GitHub repo](https://github.com/ipfs/team-mgmt).
+Learn about the ongoing work to design the predicates that enable Filecoin by exploring our [Filecoin Research website](https://research.filecoin.io/).
```

Comment:
Is there a word other than `predicates` that would work here? I suspect many folks  only know one meaning of this word:
🎶 _Mr. Morton is the subject of the sentence, and what the predicate says, he does._ 🎶

Reply:
Rewrote the sentence entirely :)